lib/elixir/lib/gen_server.ex

```diff
@@ -159,6 +159,36 @@ defmodule GenServer do
   the GenServer callbacks as doing so will cause the GenServer to misbehave.
   If you want to receive custom messages, always receive them in `handle_info/2`.
 
+  Custom messages are all other messages a server may receive that are not sent via
+  `call/3` or `cast/2`.
+
+  To send custom message use: `Kernel.send/2`,`Process.send_after/3` functions.
```

Comment:
On a second read, I'm not sure that we need to mention this at all. Maybe we can say, in the previous sentence:

> Custom messages are all other messages a server may receive that are not sent via `call/3` or `cast/2` (for example, sent via `Kernel.send/2`).

As a beginner, would you feel that's helpful enough?